.travis.yml

```diff
@@ -29,9 +29,9 @@ before_script:
   - if [ $ES_COMPOSER_NODEV == "no" ] && ! $(phpenv version-name | grep -q '5.3'); then sudo composer require "guzzlehttp/guzzle" "4.2.*" --dev --no-ansi --no-progress --no-interaction; fi
 
 script:
+  - phpunit -c test/ --coverage-clover build/coverage/clover-unit.xml --group unit
```

Comment:
Just FYI, if `unit` tests will be placed before `shutdown`, they potentially can hit ES server, that obviously not thing `unit` test is supposed to do.